python/dgl/graphbolt/itemset.py

```diff
@@ -97,7 +117,7 @@ def __init__(
         items: Union[int, Iterable, Tuple[Iterable]],
```

Comment:
```suggestion
        items: Union[int, torch.Tensor, Iterable, Tuple[Iterable]],
```
Is it necessary to specify that torch.Tensor(scalar) is supported?